src/main/java/org/jabref/gui/BasePanel.java

```diff
@@ -585,7 +585,8 @@ public void update() {
                 .openConsole(frame.getCurrentBasePanel().getBibDatabaseContext().getDatabaseFile().orElse(null)));
 
         actions.put(Actions.PULL_CHANGES_FROM_SHARED_DATABASE, (BaseAction) () -> {
-            DatabaseSynchronizer dbmsSynchronizer = frame.getCurrentBasePanel().getBibDatabaseContext()
+            DatabaseSynchronizer dbmsSynchronizer = frame.getCurrentBasePanel()
+                    .getBibDatabaseContext()
```

Comment:
I know, I'm getting on your nerves with formatting issues, but the dots should be aligned here, shouldn't they?

Reply:
That's not that easily possible as I tried to explain in the other posts. There is no simple align on dot option in Eclipse. And aligning on column will sometimes bee to agressive.